### PR TITLE
Remove warnings during tests

### PR DESCRIPTION
**Pull request by wd60622**
## Related Issue

- [ ] Closes #
- [ ] Related to #

## Checklist

- [ ] Checked that [the pre-commit linting/style checks pass](https://docs.pymc.io/en/latest/contributing/python_style.html)
- [ ] Included tests that prove the fix is effective or that the new feature works
- [ ] Added necessary documentation (docstrings and/or example notebooks)
- [ ] If you are a pro: each commit corresponds to a [relevant logical change](https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes)

## Modules affected

- [ ] MMM
- [ ] CLV

## Type of change

- [ ] New feature / enhancement
- [ ] Bug fix
- [ ] Documentation
- [x] Maintenance
- [ ] Other (please specify):

----
📚 Documentation preview 📚: https://pymc-marketing--823.org.readthedocs.build/en/823/

### CODE REVIEW

**Comment by wd60622** [comment]
WIP. The more the merrier

**Comment by juanitorduz** [comment]
Great 🙌

**Comment by wd60622** [comment]
Only 7 warnings on the 3.10 tests now. Mainly from the CLV fits with potentials
Any thoughts on this? 
https://github.com/pymc-labs/pymc-marketing/actions/runs/9876739390/job/27276622994

**Comment by wd60622** [comment]
Bunch of warnings from pytest deprecations upcoming in python 3.14...

File: pymc_marketing/mmm/delayed_saturated_mmm.py [hunk 1]
```diff
@@ -78,17 +79,17 @@ def __init__(
...
             ..., description="Type of adstock transformation to apply."
         ),
         saturation: str | InstanceOf[SaturationTransformation] = Field(
             ..., description="Type of saturation transformation to apply."
         ),
+        adstock_max_lag: int | None = Field(
+            None,
+            gt=0,
+            description="Number of lags to consider in the adstock transformation.",
+        ),
```
Comment (juanitorduz) [inline pymc_marketing/mmm/delayed_saturated_mmm.py:92]: If we have a default of `None` we must ensure in the docstrings and the Field description what is the max lag used. From what I see below is `10` but this has to be explicit for the user in the docs :)

**Review by juanitorduz — CHANGES_REQUESTED** [review]
LGTM. Just a suggestion about the docs :)

**Review by juanitorduz — APPROVED** [review]

**Pull request merged**